src/transport/PeerConnections.h

```diff
@@ -326,7 +326,7 @@ class PeerConnections
      *
      */
     template <class T>
-    void SetConnectionExpiredHandler(void (*handler)(const PeerConnectionState &, T *), T * param)
+    void SetConnectionExpiredHandler(void (*handler)(PeerConnectionState &, T *), T * param)
```

Comment:
Why the `const` removal here?  Seems to me that `OnConnectionExpired` can take a `const PeerConnectionState *` as the first arg.  Or maybe even better, since the callers are passing references anyway, it can take `const PeerConnectionState &`.

Reply:
To make it consistent with other APIs, or we can change other APIs to `const`. Which way do you prefer ?

Reply:
I'd prefer that everything that can be const be const.  If the other APIs actually need to change the `PeerConnectionState` and hence need to take a non-const, that's fine and represents a semantic difference between the APIs that we should express in the type system.  Consistency in that dimension (that is, hiding the semantic difference) seems like an anti-goal to me.

Reply:
And again, the changes to other APIs would have been best done as a separate PR.

Reply:
Thanks, I have reverted it back. And I'll submit the refactor as another PR.